test/specs/commonmark/commonmark-spec.js

```diff
@@ -169,7 +176,7 @@ describe('CommonMark 0.28 HTML blocks', function() {
   var section = 'HTML blocks';
 
   // var shouldPassButFails = [];
-  var shouldPassButFails = [132, 126, 147, 121, 124, 120, 153, 133, 119, 127, 117, 118, 141, 116, 158, 122, 123, 128, 143, 130, 136, 137, 140, 125, 134, 131, 144, 145, 146, 148, 139, 149, 129, 156, 135, 138, 142, 150, 151, 152, 154, 155, 157];
+  var shouldPassButFails = [132, 126, 147, 124, 120, 153, 133, 119, 127, 118, 141, 116, 158, 123, 143, 130, 137, 140, 125, 134, 131, 144, 145, 148, 139, 149, 129, 156, 135, 138, 155];
 
```

Comment:
What change to the lib has been made to make them pass? Honestly curious.

Reply:
I think just resetting the defaults